### PR TITLE
Feature liq 910 filter positions

### PR DESCRIPTION
**Pull request by PeGilo**

### CODE REVIEW

File: src/Lykke.Service.LiquidityEngine/Migration/StorageMigrationService.cs [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+                }
+            }
+
+            if (versionNumber != systemVersion.VersionNumber)
+            {
+                systemVersion.VersionNumber = versionNumber;
+
+                _log.InfoWithDetails("Updating storage version", systemVersion);
+
+                await _versionControlRepository.UpdateAsync(systemVersion);
```
Comment (aushakov) [inline src/Lykke.Service.LiquidityEngine/Migration/StorageMigrationService.cs]: Save version number each time migration operation successfully completed.

File: src/Lykke.Service.LiquidityEngine/Migration/StorageMigrationService.cs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                    throw new InvalidOperationException("Migration operations have inconsistent updated versions.");
+                }
+
+                return next;
+            });
+
+            return operations;
+        }
+
+        private class MigrationOperationDescription
```
Comment (aushakov) [inline src/Lykke.Service.LiquidityEngine/Migration/StorageMigrationService.cs]: Remove migration attribute and move properties to the interface. This class no needed the operation should describe is self.

File: src/Lykke.Service.LiquidityEngine.AzureRepositories/VersionControl/VersionControlRepository.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        public VersionControlRepository(INoSQLTableStorage<SystemVersionEntity> storage)
+        {
+            _storage = storage;
+        }
+
+        public async Task<SystemVersion> GetAsync()
+        {
+            SystemVersionEntity entity = await _storage.GetDataAsync(GetPartitionKey(), GetRowKey());
+
+            return entity != null
```
[the remark this answers is not in the document]
Comment (PeGilo) [reply src/Lykke.Service.LiquidityEngine.AzureRepositories/VersionControl/VersionControlRepository.cs]: Moved.

File: src/Lykke.Service.LiquidityEngine.AzureRepositories/VersionControl/VersionControlRepository.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+                ? Mapper.Map<SystemVersion>(entity)
+                : new SystemVersion { VersionNumber = 0 };
+        }
+
+        public async Task UpdateAsync(SystemVersion systemVersion)
```
[the remark this answers is not in the document]
Comment (PeGilo) [reply src/Lykke.Service.LiquidityEngine.AzureRepositories/VersionControl/VersionControlRepository.cs]: Renamed.

File: src/Lykke.Service.LiquidityEngine/Migration/StorageMigrationService.cs (hunk 1 above)
Comment (PeGilo) [reply src/Lykke.Service.LiquidityEngine/Migration/StorageMigrationService.cs]: Done.

File: src/Lykke.Service.LiquidityEngine/Migration/StorageMigrationService.cs (hunk 2 above)
Comment (PeGilo) [reply src/Lykke.Service.LiquidityEngine/Migration/StorageMigrationService.cs]: Removed attribute.

**Pull request merged**